src/crypto/store/base.js

```diff
@@ -9,3 +9,23 @@
  *
  * @interface CryptoStore
  */
+
+/**
+ * Represents an outgoing room key request
+ *
+ * @typedef {Object} OutgoingRoomKeyRequest
+ *
+ * @property {string} requestId    unique id for this request
+ *
+ * @property {string?} cancellationRequestId
+ *    transaction id for the cancellation, if any
```

Comment:
Terminology: The variable states its a request ID, but the comment says it's a transaction ID. Is there a difference? If not, can we stick to one term throughout please.